pom.xml

```diff
@@ -1675,6 +1682,20 @@
       </build>
     </profile>
 
+    <!-- use org.openlabtesting.leveldbjni on aarch64 platform except MacOS -->
+    <profile>
+      <id>aarch64</id>
+      <activation>
+        <os>
+          <family>linux</family>
+          <arch>aarch64</arch>
+        </os>
+      </activation>
+      <properties>
+        <leveldbjni.group>org.openlabtesting.leveldbjni</leveldbjni.group>
```

Comment:
is it a super set of original leveldbjni? if so, may we can switch to it unconditionally

Reply:
@pan3793, I'm not sure if it's a super set of original leveldbjni. To be on the safe side, it’s better to activate this profile based on OS settings automatically.

Reply:
I prefer to keep building script simple. don't forget to update sbt script